src/group_theory/quotient_group.lean

```diff
@@ -341,8 +352,8 @@ monoid_hom_ext _ $ monoid_hom.ext $ λ g, congr_arg coe $ h g
 isomorphism."]
 def equiv_quotient_zpow_of_equiv (χ : G' ≃* H') (n : ℤ) :
```

Comment:
Can you add `equiv_quotient_zpow_of_equiv_refl`, and similar statements for `.symm` and `.trans`?

Reply:
Do you mean this?
```
def equiv_quotient_zpow_of_equiv_symm (n : ℤ)
  (h : G' ⧸ (zpow_group_hom n : G' →* G').range ≃* H' ⧸ (zpow_group_hom n : H' →* H').range) :
  H' ⧸ (zpow_group_hom n : H' →* H').range ≃* G' ⧸ (zpow_group_hom n : G' →* G').range :=
h.symm
```
Is this better off written by the user of `equiv_quotient_zpow_of_equiv` rather than in here?

Reply:
No, I mean the lemma that `(equiv_quotient_zpow_of_equiv n e).symm = equiv_quotient_zpow_of_equiv n e.symm`